dist/scss/wds-components/_global-navigation.scss

```diff
@@ -402,4 +408,80 @@
 		margin: 6px;
 		text-align: center;
 	}
+
+	&__community-bar {
+		display: flex;
+		align-items: center;
+		position: absolute;
+		left: 0;
+		top: 0;
+		height: 100%;
+
+		@media #{$wds-breakpoint-medium-up} {
+			display: none;
+		}
+	}
+
+	&__logo-heart {
+		background-color: #002a33;
+		height: 100%;
+		width: 27px;
+		padding: 0 0 0 12px;
+		position: relative;
+		z-index: 1;
+	}
+
+	&__community-bar-separator {
+		border-left: 1px solid $wds-fandom-color-dark-gray;
+		opacity: 0;
+		height: 30px;
+		margin-left: 12px;
+		width: 1px;
+	}
+
+	.wds-community-bar__sitename {
+		color: #fff;
+		max-width: calc(100vw - 120px);
```

Comment:
I assume this 120px is the width of icons on the left and right side. Can we calculate it using sass variables?